clic/src/tier1/cleDetectMinimaKernel.cpp

```diff
@@ -0,0 +1,28 @@
+
+#include "cleDetectMinimaKernel.hpp"
+
+namespace cle
+
+{
+DetectMinimaKernel::DetectMinimaKernel(const ProcessorPointer & device)
+  : Operation(device, 2)
+
+{
+  std::string cl_header = {
+#include "cle_detect_minima.h"
+  };
+  this->SetSource("detect_minima", cl_header);
+}
```

Comment:
Library was updated, this need to be changed as followed:
- move the include outside the function, at the top of the file
- replace `cl_header` by `oclKernel::detect_minima`

see: [absolute kernel](https://github.com/clEsperanto/CLIc_prototype/blob/master/clic/src/tier1/cleAbsoluteKernel.cpp) for example